Makefile

```diff
@@ -17,7 +17,8 @@
 
```

Comment:
This makefile has un runable rules like config, purge and %.deb either fix them if requiered or remove them